examples/api/lib/material/elevated_button/elevated_button.1.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for using [ElevatedButton.iconAlignment] parameter.
```

Comment:
Looks like you chose ElevatedButton for this example. However, `iconAlignment` is also available in all buttons.

Reply:
Actually wanted replicate the same for other button if elevated one is looking fine.